Updates to README.md (#149)
returntocorp/semgrep-network-broker: README.md

```yaml
inbound:
  httpClient:

>     additionalCACerts: # Optional, useful if you have self-signed certificates. Certificates here will be appended to the Root CA trust of the container.

Reviewer: Nit: Useful sounds like you may never need to use this. Instead could we say something clearer like "necessary if you have self-signed cerftificates"?

>     additionalCACerts: # Optional, useful if you have self-signed certificates. Certificates here will be appended to the Root CA trust of the container.

Author: ```suggestion
    additionalCACerts: # Optional. Certificates here will be appended to the Root CA trust of the container. Necessary when the SCM(s) the broker interacts with have self-signed certificates.
```

      - /path/to/custom/cert.pem
    tlsMinVersion: "1.2" # Optional. Valid values: "1.2", "1.3". Defaults to "1.3" if unset.
```

An alternative to stipulating `additionalCACerts:` is setting the `$SSL_CERT_DIR` environment variable at time of container creation.

> Example:

Reviewer: Nit: Can we briefly explain what the following command does, e.g. it mounts the certs dir into the container, just in case the reader isn't familiar with Docker commands?

Author: WDYT about the edit here? My edit's a little wordy. Could pare down?

```bash

> docker run \

Author: ```suggestion
$ docker run \
```

  ...

>   -v /path/containing/your/certs:/certs \

Author: ```suggestion
  -v /path/containing/your/certs:/certs \ # mount a path from the host machine as a container volume
```

>   -e SSL_CERT_DIR=/certs \

Author: ```suggestion
  -e SSL_CERT_DIR=/certs \ # set the $SSL_CERT_DIR environment variable to the mounted volume
```
